src/ModelRenderer.js

```diff
@@ -73,19 +120,42 @@ function ModelRenderer({ cameraPosition, onOrbitChange, ...props }) {
       />
       <ambientLight />
       <Suspense fallback={null}>
-        <Model path="models/gltf/nike_shoe/scene.gltf" />
+        <Model
+          path="models/gltf/nike_shoe/scene.gltf"
+          onPointerMove={pinFollowCursor && onPointerMove}
```

Comment:
Is there a reason for introducing the new stateful `pinFollowCursor` instead of using `cursor === CURSOR.Pin`?